clients/fides-js/src/components/Overlay.tsx

```diff
@@ -150,9 +142,6 @@ const Overlay: FunctionComponent<Props> = ({
 
   return (
     <div>
-      {bannerIsOpen && window.Fides.options.preventDismissal && (
-        <div className="fides-modal-overlay" />
```

Comment:
could you explain this change a bit more? The only functionality we want to keep for dismissals is that if `Fides.options.preventDismissal` is `true`, then we should remove the "x" from the banner / modal, otherwise only allow dismissals through that "x", correct?

Reply:
didn't mean to remove